PHPCI/Helper/BaseCommandExecutor.php

```diff
@@ -57,10 +55,9 @@
      * @param bool        $quiet
      * @param bool        $verbose
      */
-    public function __construct(BuildLogger $logger, $rootDir, &$quiet = false, &$verbose = false)
+    public function __construct(BuildLogger $logger, $rootDir, &$verbose = false)
```

Comment:
@Adirelle I'm pretty sure that having ```$verbose``` param passed by reference is not required/useful.